### PR TITLE
Fix crashes due to un-initialized GSP shared memory

### PR DESCRIPTION
**Pull request by yuriks**
Fixes crashes in MSVC debug mode introduced by #725.

### CODE REVIEW

File: src/core/hle/service/gsp_gpu.cpp [hunk 1]
```diff
@@ -31,12 +31,11 @@ Kernel::SharedPtr<Kernel::Event> g_interrupt_event;
 /// GSP shared memoryings
 Kernel::SharedPtr<Kernel::SharedMemory> g_shared_memory;
 /// Thread index into interrupt relay queue, 1 is arbitrary
```
Comment (Subv) [inline src/core/hle/service/gsp_gpu.cpp]: The comment is now out of sync :)

Comment (yuriks) [inline src/core/hle/service/gsp_gpu.cpp]: asdf

**Comment by archshift** [comment]
LGTM

**Comment by bunnei** [comment]
LGTM

**Comment by archshift** [comment]
```
[18:06:28]  <@bunnei>	merge it !
[18:06:33]  <JMC47>	merge it
[18:06:37]  <Lioncash>	merge it
[18:07:09]  <@archshift>	merge it?
```

**Pull request merged**